spec/models/restaurant_spec.rb

```diff
@@ -7,6 +7,11 @@
     it { is_expected.to have_db_column :address }
     it { is_expected.to have_db_column :email }
     it { is_expected.to have_db_column :phone }
+    it { is_expected.to have_db_column :city  }
+    it { is_expected.to have_db_column :state  }
+    it { is_expected.to have_db_column :country  }
+    it { is_expected.to have_db_column :latitude }
+    it { is_expected.to have_db_column :longitude }
     it { is_expected.to have_db_column :description }
   end
 
```

Comment:
IMO you need unit specs for `full_address` and `address_city_state_country_present?` and `address_city_state_country_changed?` (unless you make the last ones `private`)